pkg/pki/k8scsrpki/k8scsr_user.go

```diff
@@ -309,3 +320,27 @@ func isKafkaUserCertificateReady(secret *corev1.Secret, includeJKS bool) bool {
 
 	return true
 }
+
+// Approve approves certificate signing requests
+func (c *k8sCSR) Approve(ctx context.Context, signingReq *certsigningreqv1.CertificateSigningRequest) error {
+	cond := certsigningreqv1.CertificateSigningRequestCondition{
+		Type:    certsigningreqv1.CertificateApproved,
+		Status:  corev1.ConditionTrue,
+		Reason:  approveReason,
+		Message: "CSR has been approved by signer",
```

Comment:
I am a bit confused about this message - Koperator is not the signer here, right? My understanding was that:
1. Koperator approves CSR
2. The actual signer (in this case, cert-manager) detects CSR gets approved by the approver, it then signs the CSR

Reply:
Is it OK to replace this with "CSR has been approved by kafka-operator"?

Reply:
I think we should either use `"Koperator"` or just `"the operator"` instead of `"kafka-operator"` since this is a deprecated name of the Koperator

Reply:
Ok, thanks. `"Koperator"` is the best option because some other tools can also theoretically approve CSRs.